web/src/containers/P2P/TradesHistory/index.tsx

```diff
@@ -120,8 +122,8 @@ const TradesHistory: FC = (): ReactElement => {
     return (
         <div>
             {fetching && <div className="text-center"><Spinner animation="border" variant="primary" /></div>}
```

Comment:
Create a separate method using useMemo for this block

Reply:
Fixed, thanks.